gradle/detect.go

```diff
@@ -14,58 +14,90 @@
  * limitations under the License.
  */
 
-package gradle
+ package gradle
```

Comment:
Everything in this file is indented over one space. It's throwing off the diff and making it look like the whole file was changed.

Please run `gofmt` on the file & push that up. Some editors will do this automatically, like VS Code. Goland doesn't but you can make it do that (Preferences -> Actions on Save or Preferences -> File Watchers).

Reply:
I used VS Code (CodeSpaces). If that's not working maybe you could put a `.devcontainer` in the repo so others will find it easier?

Reply:
I don't honestly know anything about CodeSpaces or `.devcontainer`, but it's something I can check out and see if we could enable.

If you're using VSCode, all you need is the [Go extension](https://code.visualstudio.com/docs/languages/go#_formatting) and it should just work.

Reply:
It seems like a trivial change for you, and I’m a newbie, so there is a learning curve. Do you really want me to make it, or can you handle it when you merge?

Reply:
FWIW I am using the Go extension - I don't think you need a .devcontainer for that to wqork by default. But when I run `gofmt` it never returns (waiting for input on stdin maybe?).

Reply:
`go fmt ./gradle`

(Not at all obvious IMHO.)